demisto_sdk/commands/common/content/tests/objects/pack_objects/pack_metadata/pack_metadata_test.py

```diff
@@ -805,87 +804,3 @@ def mock_pack_metadata_for_data_source(support, default_data_source, integration
     content_items = PackContentItems()
     content_items.integration.extend(integrations)
     return content_items, my_instance
-
-
-@pytest.mark.parametrize(
```

Comment:
why just remove this test?

Reply:
I reverted my previous version + added some new handling on the new so I deleted the tests from my previous version.